tests/acceptance/features/apiCors/cors.feature

```diff
@@ -69,3 +69,41 @@ Feature: CORS headers
       | ocs_api_version | endpoint                          |
       | 1               | /apps/files_sharing/api/v1/shares |
       | 2               | /apps/files_sharing/api/v1/shares |
+
+
+  Scenario: CORS headers should be returned when setting CORS domain sending origin header in the Graph api
+    When user "Alice" lists all available spaces with headers using the Graph API
+      | header | value               |
+      | Origin | https://aphno.badal |
+    Then the HTTP status code should be "200"
+    And the following headers should be set
+      | header                      | value               |
+      | Access-Control-Allow-Origin | https://aphno.badal |
+
+  @issue-8231
+  Scenario: CORS headers should be returned when setting CORS domain sending origin header in the Webdav api
+    When user "Alice" sends PROPFIND request to space "Alice Hansen" with headers using the WebDAV API
+      | header | value               |
+      | Origin | https://aphno.badal |
+    Then the HTTP status code should be "207"
+    And the following headers should be set
+      | header                      | value               |
+      | Access-Control-Allow-Origin | https://aphno.badal |
+
+
+  Scenario Outline: CORS headers should be returned when setting CORS domain sending origin header in the OCS api
+    Given using OCS API version "<ocs_api_version>"
+    When user "Alice" sends HTTP method "GET" to OCS API endpoint "<endpoint>" with headers
+      | header | value               |
+      | Origin | https://aphno.badal |
```

Comment:
This scenario is same as https://github.com/owncloud/ocis/blob/76511600db1275d0a046f09c0e5bbcfb383f472b/tests/acceptance/features/apiCors/cors.feature#L13

Reply:
Yes, I know. but it skipped because https://github.com/owncloud/ocis/issues/5195
partially this test may work so I added it

Reply:
maybe then we should add the comment somewhere so that when the issue is fixed we can remove this scenario